src/main/java/com/team1816/frc2020/subsystems/Shooter.java

```diff
@@ -121,6 +133,16 @@ public void setVelocity(double velocity) {
         outputsChanged = true;
     }
 
+    public void startShooterBasedOnDistance() {
+        /* if (distance <  ) { TODO: Find correct distances
+            setVelocity(MID_VELOCITY);
+        } else if (< distance < ) {
+            setVelocity(NEAR_VELOCITY);
```

Comment:
`MID_VELOCITY` and near velocity need to be swapped.
```suggestion
            setVelocity(NEAR_VELOCITY);
        } else if (< distance < ) {
            setVelocity(MID_VELOCITY);
```